run-in-vm.sh

```diff
@@ -1,30 +1,33 @@
 #!/bin/bash
-set -euo pipefail
+set -x -euo pipefail
 name=${VMNAME:-backend-server}
-test -x "$(command -v multipass)" || { echo "You need multipass installed to run this script: https://multipass.run"; exit 1; }
+test -x "$(command -v multipass)" || { echo "You need multipass installed to run this script: https:/multipass.run"; exit 1; }
 
 delete() {
     echo "Removing $name..."
-    multipass delete --purge "$name" >/dev/null 2>&1 || true &
+    multipass delete --purge "$name" >/dev/null 2>&1 || true
 }
 
 
 # always start clean
 multipass delete --purge "$name" 2>/dev/null || true
 
 multipass launch 20.04 --name "$name"
-trap delete EXIT
+test -z "${KEEP:-}" && trap delete EXIT
 
 # add this directory in VM as /host, with current user mapped to root
```

Comment:
update doc to reflect code change? (removal of `--uid-map`)

Reply:
Will do